internal/store/horizontalpodautoscaler.go

```diff
@@ -184,6 +184,68 @@ func hpaMetricFamilies(allowAnnotationsList, allowLabelsList []string) []generat
 				return &metric.Family{Metrics: ms}
 			}),
 		),
+		*generator.NewFamilyGenerator(
+			"kube_horizontalpodautoscaler_status_current_metric",
+			"The current value of the metric defined in the spec section.",
```

Comment:
```suggestion
			"The current value of the metric defined in the spec.",
```

Reply:
no problem. updated.
Also fixed CI issue, did not realize that `make container` did not run them.
ty